docs/getting-started.md

```diff
@@ -157,49 +185,56 @@ excludes:
-- Do not bother manually sorting path or scope excludes entries in the `ort.yml` file, simply add them and use `orthw rc-sort` to sort the file
+- Do not sort path or scope excludes entries in the `ort.yml` file manually, but simply add them and use `orthw rc-sort` to sort the file
 
 ### Example .ort.yml files
 
 - [The .ort.yml of harp.gl][ort-yml-harp.gl], a 3D web map rendering engine written in TypeScript
 - [The .ort.yml of MoveTK][ort-yml-movetk], a c++ library for computational movement analysis 
 - [The .ort.yml of XYZ Hub][ort-yml-xyz-hub], a RESTful web service for the access and management of geospatial data written in Java
 
-## Checking your Project Dependencies for Security Advisories <a name="orthw-check-advisories"></a>
+## Checking Project Dependencies for Security Advisories <a name="orthw-check-advisories"></a>
 
 ORT supports querying multiple security advisory providers to help you safely use open source software in your project.
 
 The `~/.orthwconfig` file is preconfigured to use [OSV][osv] as the default advisory provider but you are free to add one or more other services such as [OSS Index][oss-index] or your local instance of [VulnerableCode][vulnerablecode].
 
-Run the below command to check your project dependencies for known security vulnerabilities:
+Run the command below to check your project dependencies for known security vulnerabilities:
 
 ```bash
 cd ~/ort-scans/mime-types-orthw
 orthw check-advisories
 ```
 
-[Generate the Web App report](#orthw-report-webapp) using `orthw report-webapp` and it should show several security vulnerabilities and increase in policy violations under _Summary_.
+[Generate the Web App report](#orthw-report-webapp) using `orthw report-webapp` and it should show several security
+_Vulnerabilities_ and an increased number of policy violations under _Summary_.
 
 ## Correcting Invalid or Missing Package Metadata <a name="orthw-curations"></a>
 
-Using [curations][ort-curations] you can correct invalid or missing package metadata including the declared license and code repository information.
+You can use [curations][ort-curations] to add or correct missing or invalid package metadata, including the declared
+license and code repository information. 
 
-In the WebApp report look up `NPM::acorn:7.1.1` by filtering the _Package_ column under _Table_. You will find the declared license is MIT but the detected are Apache-2.0, BSD-2-Clause, BSD-3-Clause, LicenseRef-scancode-facebook-patent-rights-2 and MIT, so which licenses apply?
+In the WebApp report, look up `NPM::acorn:7.1.1` by filtering the _Package_ column under _Table_. Notice that the
+declared license is MIT, but ORT detected Apache-2.0, BSD-2-Clause, BSD-3-Clause,
+LicenseRef-scancode-facebook-patent-rights-2 and MIT, so which licenses apply? 
 
-### Determining whether Right Sources where Scanned
+### Determining whether Right Sources Were Scanned
```

Comment:
Just double checking: capizalized-first is intended?

Reply:
Yes, it is. It's called the "title case" in English and the simple rule is that all the significant words are capitalised and others are lower case. The very first letter in the title should be capital unless it is a fixed name and then all nouns, verbs, adjectives, but not pronouns (e.g. he, she, it), prepositions (.e.g. at, in, etc.), articles (e.g. the, a(n)), conjunctions (e.g. and, if, also, whether, etc.)

For a further example, if you have a name that is always written in lower case, such as the command 'ls', a title with it at the beginning would be like this: "`ls` and its Switches". I know it looks strange, but that's how it is. (To avoid a title looking strange it's best to change it sometimes).

Reply:
Thanks!